services/thumbnails/README.md

```diff
@@ -0,0 +1,58 @@
+# Thumbnails Service
+
+The thumbnails service provides methods to generate thumbnails for various files and resolutions based on requests. It retrieves the sources at the location where the user files are stored and saves the tumbnails where system files are stored. Those locations have defaults but can be manually defined via environment variables.
+
+## File Locations Overview
+
+The relevant environment variables defining file locations are:
+
+-   (1) `OCIS_BASE_DATA_PATH`
+-   (2) `STORAGE_USERS_OCIS_ROOT`
+-   (3) `THUMBNAILS_FILESYSTEMSTORAGE_ROOT`
+
+(1) ... Having a default set by the ocis code, but if defined, used as base path for other services.  
+(2) ... Source files, defaults to (1) plus path component, but can be freely defined if required.  
+(3) ... Target files, defaults to (1) plus path component, but can be freely defined if required.  
+
+For details and defaults for these environment variables see the ocis admin documentation.
+
+## Thumbnail Location
+
+It may be benificial to define the location of the thumbnails other than at system files. This is due the fact that storing thumbnails can consume a lot of space over time which not necessarily needs to reside on the same partition or mount or expensive drives.
```

Comment:
```suggestion
It may be beneficial to define the location of the thumbnails to be other than the default (with system files). This is due the fact that storing thumbnails can consume a lot of space over time which not necessarily needs to reside on the same partition or mount or expensive drives.
```